src/uu/df/src/table.rs

```diff
@@ -18,10 +18,14 @@ use uucore::fsext::{FsUsage, MountInfo};
 use std::fmt;
 use std::ops::AddAssign;
 
+type InodesIntT = u128;
+type MaybeInodesT = Option<InodesIntT>;
```

Comment:
I appreciate the idea here, but I think it obfuscates the exact data type too much. Id rather just see `Option<u128>` everywhere than `MaybeInodesT` because then I actually know what I'm dealing with.